src/components/InForm/InForm.tsx

```diff
@@ -85,6 +76,22 @@ export const InForm: React.FunctionComponent<any> = (props) => {
     IComboBoxOption[]
```

Comment:
gradeRankOptions should be a const rather than state (it changes indirectly, based on the empType selection, and can be determined at render what those options should be).

Reply:
Good call,  Tracking via new issue #15